backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceCandDAO.java

```diff
@@ -464,7 +465,22 @@ public List<I_C_InvoiceCandidate_InOutLine> retrieveICIOLAssociationsFor(@NonNul
 				.create()
 				.list(I_C_InvoiceCandidate_InOutLine.class);
 	}
-	
+
+	public boolean isInvoiceAlreadyCreated(@NonNull final InOutId inOutId)
```

Comment:
this needs to make sure that the invoice in question is completed or closed (not reversed, drafted etc)